DBMS/DBMS-Session5/Assingment_3/UpdateOrderStatus.java

```diff
@@ -0,0 +1,25 @@
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class UpdateOrderStatus {
+
+	/**
+	* @return - Total rows affected
+	*/
+	public static int setproductInActive() {
```

Comment:
Question was to delete all those products which were not ordered by any Shopper in last
1 year, not to update.